lib/compress/index.js

```diff
@@ -775,7 +773,7 @@ AST_Scope.DEFMETHOD("hoist_declarations", function(compressor) {
                     if (
                         hoist_vars
                         && node instanceof AST_Var
-                        && !(node instanceof AST_Using || node instanceof AST_AwaitUsing)
+                        && !(node instanceof AST_Using)
```

Comment:
Is this check required? It seems to me `node instanceof AST_Var` already implies `!(node instanceof AST_Using)`.